### PR TITLE
[Azure AD roles] Devices In AU support [beta]

### PR DESCRIPTION
**Pull request by DougKirschner**

### CODE REVIEW

File: changelog/Microsoft.DirectoryServices.json [hunk 1]
```diff
@@ -1,6 +1,56 @@
...
    {
+    "ChangeList": [
+      {
+        "Id": "e34aceeb-8c4b-430a-ab0e-521201013853",
+        "ApiChange": "Method",
+        "ChangedApiName": "memberOf",
+        "ChangeType": "Addition",
+        "Description": "Added the [administrative units](https://docs.microsoft.com/en-us/graph/api/device-list-memberof.md) as a supported return type.",
+        "Target": "Device list memberOf"
+      },
```
Comment (FaithOmbongi) [inline changelog/Microsoft.DirectoryServices.json:12]: Proposed change
```suggestion
      {
        "Id": "e34aceeb-8c4b-430a-ab0e-521201013853",
        "ApiChange": "Method",
        "ChangedApiName": "memberOf",
        "ChangeType": "Addition",
        "Description": "Added `administrativeUnit` as a supported return type of the [List memberOf](https://docs.microsoft.com/en-us/graph/api/device-list-memberOf?view=graph-rest-beta) method of the device resource.",
        "Target": "memberOf"
      },
```

File: changelog/Microsoft.DirectoryServices.json [hunk 2]
```diff
@@ -1,6 +1,56 @@
...
+        "Target": "Administrative unit delete members"
+      },
+      {
+        "Id": "e34aceeb-8c4b-430a-ab0e-521201013853",
+        "ApiChange": "Method",
+        "ChangedApiName": "members",
+        "ChangeType": "Addition",
+        "Description": "Added devices as a supported resource type for administrative unit method for [list member](https://docs.microsoft.com/en-us/graph/api/administrativeunit-list-members).",
+        "Target": "Administrative unit list members"
+      }
```
Comment (FaithOmbongi) [inline changelog/Microsoft.DirectoryServices.json:44]: Proposed change. Replacing with a single object to cover all CRUD methods.
```suggestion
      {
        "Id": "e34aceeb-8c4b-430a-ab0e-521201013853",
        "ApiChange": "Method",
        "ChangedApiName": "device",
        "ChangeType": "Addition",
        "Description": "Added `device` as a supported resource type of the [Get member](https://docs.microsoft.com/en-us/graph/api/administrativeunit-get-members), [List members](https://docs.microsoft.com/en-us/graph/api/administrativeunit-list-members?view=graph-rest-beta), [Add member](https://docs.microsoft.com/en-us/graph/api/administrativeunit-post-members?view=graph-rest-beta), and [Remove member](https://docs.microsoft.com/en-us/graph/api/administrativeunit-delete-members?view=graph-rest-beta) methods of the [administrativeUnit](https://docs.microsoft.com/en-us/graph/api/resources/administrativeunit?view=graph-rest-beta) resource.",
        "Target": "administrativeUnit"
      }
```

**Review by FaithOmbongi — CHANGES_REQUESTED** [review]
My apologies @DougKirschner , this slipped from my mind.

- Shared a proposed change for the changelog.
- Also, update the [administrativeUnit](https://docs.microsoft.com/en-us/graph/api/resources/administrativeunit?view=graph-rest-beta) resource and add references to devices in the opening line, description of methods, and description of relationships.
- Add a reference to administrativeUnits in the [device](https://docs.microsoft.com/en-us/graph/api/resources/device?view=graph-rest-beta) resource: List memberOf description in the methods table, and the memberOf relationship.

**Comment by FaithOmbongi** [comment]
@DougKirschner pushed a few commits to clean up other areas in docs & update the changelog date.

**Review by FaithOmbongi — APPROVED** [review]

**Comment by DougKirschner** [comment]
Hi @FaithOmbongi putting this one back to "do not merge" so we can release simultaneously with the other device documents in a few weeks.  While the API is technically done, we can time the release so that the Azure AD portal UX rolls out along side the API docs.

**Comment by FaithOmbongi** [comment]
@DougKirschner This is well noted.

**Comment by DougKirschner** [comment]
@FaithOmbongi  Ready to release this update as the rest of the feature is now rolling out.

**Pull request merged**